.github/workflows/generate-upload-enterprise-3p-fossa-report.yml

```
@@ -48,7 +48,9 @@ jobs:
 
       - name: Get repository name
         id: get_repo_name
-        run: echo "repo_name=${{ github.event.repository.name }}" >> $GITHUB_OUTPUT
+        env:
+          REPO_NAME: ${{ github.event.repository.name }}
+        run: echo "repo_name=$REPO_NAME" >> $GITHUB_OUTPUT
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:1:32: Double quote to prevent globbing and word splitting

(shellcheck)